core/src/main/java/org/elasticsearch/search/SearchModule.java

```diff
@@ -286,6 +286,7 @@ public SearchModule(Settings settings, NamedWriteableRegistry namedWriteableRegi
 
         registerBuiltinFunctionScoreParsers();
         registerBuiltinQueryParsers();
+        registBuiltinRescorers();
```

Comment:
I moved this here because we already do it for other things and because it makes testing just a tiny bit easier.